### PR TITLE
Update geospatial readme

### PR DESCRIPTION
**Pull request by jarrodmillman**

### CODE REVIEW

File: examples/geospatial/extended_description.rst [hunk 1]
```diff
@@ -3,6 +3,16 @@
 Geospatial Examples Description
 -------------------------------
 
+NetworkX used to provide functions for reading and writing
```
Comment (dschult) [inline examples/geospatial/extended_description.rst:6]: NX still provides these functions. And technically, the old versions of NX will continue to provide them. :} 
I know.... picky picky...  How about a lead in sentence like:
`Functions for reading and writing shapefiles are provided in NetworkX versions <3.0.`

File: examples/geospatial/extended_description.rst [hunk 2]
```diff
@@ -3,6 +3,16 @@
...
+shapefiles.  A shapefile is a geospatial vector
+data format for geographic information systems software.
+We deprecated those functions in 2.6 and they will be removed
+in 3.0.
+
+We recommend that you use the standard libraries for working
+with geospatial data in Python.  The geospatial examples show
+how to use these libraries with NetworkX.
```
Comment (dschult) [inline examples/geospatial/extended_description.rst:14]:
```suggestion
with geospatial data in Python.  The following geospatial examples show 
how to use these libraries with NetworkX.
```

**Review by dschult — APPROVED** [review]
This looks good!   You can ignore or do-what-you-like with my comments.

File: examples/geospatial/extended_description.rst (hunk 2 above)
Comment (jarrodmillman) [reply examples/geospatial/extended_description.rst]: I shortened this section.

**Pull request merged**